config/software/rbzmq.rb

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright 2012-2017 Chef Software, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name "rbzmq"
+default_version "master"
+source git: "git@github.com:chef/rbzmq.git"
```

Comment:
I /think/ we consistently use https for github origins, so it may be good to do the same here.

Reply:
Thanks, good catch. It's updated now.